src/cmd/manage_nsfs.js

```diff
@@ -14,6 +14,7 @@ const SensitiveString = require('../util/sensitive_string');
 const ManageCLIError = require('../manage_nsfs/manage_nsfs_cli_errors').ManageCLIError;
 const ManageCLIResponse = require('../manage_nsfs/manage_nsfs_cli_responses').ManageCLIResponse;
 const bucket_policy_utils = require('../endpoint/s3/s3_bucket_policy_utils');
+const mongodb = require('mongodb');
```

Comment:
As far as I recall we only use `mongodb.ObjectId` which should be a simple function to generate the ID. 
We should remove `mongodb` all togather and not do a new use for it.

Reply:
@liranmauda I asked @naveenpaul1 to use mongodb.ObjectId because of the fact that in the future we might add a db for data buckets non containerized like we had in the original non containerized design

Reply:
@romayalon I get why we need the id.
We want to remove the mongo requirement from the whole code.
all we use from this *old* module is the object ID which should be a function such as the one in: https://gist.github.com/solenoid/1372386
I did not block the PR on this comment (but did on others), but I think we should replace all `mongodb.ObjectId` in our code with a native written function (we can use the one I pointed to and write a comment to refer to it.)

Reply:
@liranmauda I could see in `postgres_client.js` we are checking `id` is an instanceof `mongodb.ObjectId`, Will it create issues if we remove `mongodb.ObjectId` completely.